one_step_setup

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+set -e
+
+function fail {
+    # Print a newline to stderr and exit.
+    echo > /dev/stderr
+    exit 1
+}
+
+
+# `$PANTS_VERSION` must be set
+if [ -z "$PANTS_VERSION" ]
+then
+    echo \`\$PANTS_VERSION\` must be set. Finish installing Pants by re-running this script \
+        with \`PANTS_VERSION\` set. > /dev/stderr
+    fail
+fi
+
+# Make sure we don't override an existing Pants installation.
+if [ -f "pants.toml" ]
+then
+    echo A \`pants.toml\` file already exists in this directory. This installation script \
+        needs to create a new \`pants.toml\` file. > /dev/stderr
+    echo If you are trying to upgrade to a newer version of Pants, set the \`pants_version\` \
+        setting in your existing \`pants.toml\` file. > /dev/stderr
+    echo If you are trying to run the install process from scratch, delete the \`pants.toml\` \
+        file and start again. > /dev/stderr
+    fail 
+fi
+
+if [ -f "pants" ]
+then
+    echo A file called \`pants\` already exists in this directory. This installation script \
+        will overwrite this file. To proceed, either rename this file, or delete the \
+        \`pants\` file. > /dev/stderr
+    fail
+fi
+
+# Create enough of a pants.toml file that our bootstrap process can run
+printf '[GLOBAL]\npants_version = "'$PANTS_VERSION'"\n' > pants.toml
```

Comment:
We probably want to add conditional logic to add certain defaults like `use_deprecated_cli_args = false`, based on the Pants version they're targeting. Fine in a followup, as long as we have a way to do it. `./pants` already had logic to parse version strings.

Reply:
I think it might be a bit more welcome now. There's a good reason to do backend selection from within Pants (because it might be a bit more self-aware), for instance.